lang/de.yml

```
@@ -41,27 +41,27 @@ HELP_EXCLUDE_2: "Beispiele: [e:stone], [e:Notch], [e:stone,Notch]"
 HELP_INCLUDE_2: "Beispiele: [i:stone], [i:zombie], [i:stone,wood,bedrock]"
-HELP_INSPECT_1: "Wenn der Inspektor aktiviert ist, können Sie:"
-HELP_INSPECT_2: "Klicken Sie mit der linken Maustaste, um die Platzierung anzuzeigen."
-HELP_INSPECT_3: "Klicken Sie mit der rechten Maustaste, um benachbarte Entfernungen anzuzeigen."
-HELP_INSPECT_4: "Platzieren Sie einen Block, um die Entfernung zu sehen."
-HELP_INSPECT_5: "Legen Sie einen Block in Flüssigkeit, um die Platzierung zu sehen."
-HELP_INSPECT_6: "Klicken Sie mit der rechten Maustaste, um Truhentransaktionen anzuzeigen."
-HELP_INSPECT_7: "Tipp: Sie können einfach \"/co i\" verwenden."
+HELP_INSPECT_1: "Mit dem Inspector aktiviert, kannst du Folgendes tun:"
+HELP_INSPECT_2: "Linksklick auf einen Block, um zu sehen, wer diesen Block platziert hat."
+HELP_INSPECT_3: "Rechtsklick auf einen Block, um zu sehen, welcher benachbarte Block zerstört wurde."
+HELP_INSPECT_4: "Einen Block platzieren, um zu sehen, welcher Block an dieser Stelle zerstört wurde."
+HELP_INSPECT_5: "Einen Block in Flüssigkeit (etc.) platzieren, um zu sehen, wer sie platziert hat."
+HELP_INSPECT_6: "Rechtsklick auf eine Tür, Truhe, etc., um zu sehen, wer sie zuletzt verwendet hat."
+HELP_INSPECT_7: "Tipp: Du kannst einfach \"/co i\" verwenden, um schnelleren Zugriff zu erhalten."
 HELP_INSPECT_COMMAND: "Schaltet den Blockinspektor um."
 HELP_LIST: "Zeigt eine Liste aller Befehle an."
 HELP_LOOKUP_1: "Befehlsverknüpfung."
 HELP_LOOKUP_2: "Weitere Seiten anzeigen."
 HELP_LOOKUP_COMMAND: "Blockdaten nachschlagen."
 HELP_NO_INFO: "Informationen zum Befehl \"{0}\" nicht gefunden."
 HELP_PARAMETER: "Bitte sehen Sie \"{0}\" für detaillierte Parameterinformationen."
-HELP_PARAMS_1: "Führen Sie die {Nachschlagen|Rollback|Wiederherstellen}."
-HELP_PARAMS_2: "Geben Sie die Benutzer an {Nachschlagen|Rollback|Wiederherstellen}."
-HELP_PARAMS_3: "Geben Sie die Zeit an, um {Nachschlagen|Rollback|Wiederherstellen}."
-HELP_PARAMS_4: "Beschränken Sie die {Nachschlagen|Rollback|Wiederherstellen} zu einem Radius."
-HELP_PARAMS_5: "Beschränken Sie die {Nachschlagen|Rollback|Wiederherstellen} zu einer Aktion."
-HELP_PARAMS_6: "Einschließen von Blöcken/Entitäten in die {Nachschlagen|Rollback|Wiederherstellen}."
-HELP_PARAMS_7: "Sperren/Benutzer von der ausschließen {Nachschlagen|Rollback|Wiederherstellen}."
+HELP_PARAMS_1: "Führe den {Lookup|Rollback|Restore} durch."
+HELP_PARAMS_2: "Gib die Benutzer an, für die der {Lookup|Rollback|Restore} durchgeführt werden soll."
+HELP_PARAMS_3: "Gib die Zeitspanne an, für die der {Lookup|Rollback|Restore} durchgeführt werden soll."
+HELP_PARAMS_4: "Gib einen Radius an, um den {Lookup|Rollback|Restore} auf einen bestimmten Bereich zu beschränken."
+HELP_PARAMS_5: "Beschränke den {Lookup|Rollback|Restore} auf eine bestimmte Aktion."
+HELP_PARAMS_6: "Schließe bestimmte Blöcke/Entitäten in den {Lookup|Rollback|Restore} ein."
+HELP_PARAMS_7: "Schließe Blöcke/Benutzer von dem {Lookup|Rollback|Restore} aus."
```

Shortened this so it fits on one line, although I somewhat prefer the old translations

```suggestion
HELP_PARAMS_1: "Führe einen {Lookup|Rollback|Restore} durch."
HELP_PARAMS_2: "Spieler, für die der {Lookup|Rollback|Restore} gemacht werden soll."
HELP_PARAMS_3: "Zeitspanne, für die der {Lookup|Rollback|Restore} gemacht werden soll."
HELP_PARAMS_4: "Radius, für den der {Lookup|Rollback|Restore} gemacht werden soll."
HELP_PARAMS_5: "Aktion, für die der {Lookup|Rollback|Restore} gemacht werden soll."
HELP_PARAMS_6: "Schließe Blöcke/Entitäten in den {Lookup|Rollback|Restore} ein."
HELP_PARAMS_7: "Schließe Blöcke/Benutzer von dem {Lookup|Rollback|Restore} aus."
```